.github/workflows/zcash-lightwalletd.yml

```diff
@@ -11,7 +11,7 @@ on:
       # rebuild lightwalletd whenever the related Zebra code changes
       #
       # TODO: this code isn't compiled in this docker image
-      #       rebuild whenever the actual code at zcash/lightwalletd/master changes
+      #       rebuild whenever the actual code at lightwalletd/master changes
```

Comment:
(I had added a comment about this but didn't publish it :smiling_face_with_tear: )

My idea was to avoid having unneeded repetition so that we don't need to change it if we ever change the repository again.